pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -1435,6 +1437,7 @@ func (nc *Controller) cancelPodEviction(node *v1.Node) bool {
 //   Returns false if the node name was already enqueued.
 // - deletes pods immediately if node is already marked as evicted.
 //   Returns false, because the node wasn't added to the queue.
+
 func (nc *Controller) evictPods(node *v1.Node, pods []*v1.Pod) (bool, error) {
 	// if node is in autonomy status, skip evict pods from the node
 	if node != nil && node.Annotations != nil && node.Annotations[nodeutil.AnnotationKeyNodeAutonomy] == "true" {
```

Comment:
Do we need to modify this file?

Reply:
Inconsistent versions of golint resulted in mistaken changes, which have been restored.